libs/features/personalization/manifest-utils.js

```diff
@@ -1,5 +1,6 @@
-import { loadLink } from '../../utils/utils.js';
+import { loadLink, getMetadata } from '../../utils/utils.js';
 
+const REGEX_PROMO = /\/promos\/(\w+)\//;
```

Comment:
is \w+ enough for first level folder? characters like '-' or '_' are allowed, no?